framework/src/plugins/base_plugin.ts

```diff
@@ -295,64 +284,58 @@ export abstract class BasePlugin<
 // TODO: Once the issue fixed we can use require.resolve to rewrite the logic
 //  https://github.com/facebook/jest/issues/9543
 export const getPluginExportPath = (
```

Comment:
Please change this function name as well matching to plugin property you renamed to `nodeModulePath`